sda-download/biohackathon/paper.md

```diff
@@ -151,29 +151,36 @@ sequenceDiagram
  ```
 
 
-## Extra updates
+## Additional updates
 
-Two docker compose files were developed in order to include the whole infrastructure for the sensitive data archive and the htsget-rs version. These compose files allow the users to make requests to the htsget and get the response with the parts of the data that the client then can download and assemble them.
+Two docker compose files were developed in order to include the whole infrastructure for the sensitive data archive and the htsget-rs version. These compose files allow the users to make requests to the htsget and receive a response containing pointers to the requested parts of the data, which the client can then download and assemble.
 
 The Sensitive Data Archive Command Line Interface (sda-cli) has a demo version of the htsget command.
 This command, when provided with the public key of a user and (potentially) a region for a file, it makes the request to htsget, retrieves the results, requests the particular regions from the sda-download and then assembles these parts together.
 
 
 # Conclusions and Future work
 
-Part of the members that joined our project was a member of the team which implements and develops the
+Some of the collaborators that joined our project were also members of the team developing the
 htsget protocol for bioinformatics in Rust from the University of Melbourne (Center for Cancer Research).
-This in person collaboration helped on identifying the needs and closely work together on developing the sda-download and
-htsget server on meeting those needs and moving towards achieving the project goal.
+This in person collaboration helped identifying the needs to achieve the project goal,
+and to closely work together on developing the sda-download and
+htsget server to meet those needs.
 
 The project was focused on improving the data access part of the sensitive data archive infrastructure. The existing process
 of data access was cumbersome: user receiving the whole file even when only a small part of it was needed.
 To address this, the htsget-rs was extended to support access to encrypted data and it is planned to extend it further
 by implementing the crypt4gh and enabling support for crypt4gh edit lists. The sda-download service was also
```

Comment:
what does "implementing the crypt4gh" mean here..?

Reply:
fixed